src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/events/AttachDocsToSupplementaryEvidence.java

```diff
@@ -40,15 +40,17 @@ public AttachDocsToSupplementaryEvidence(
      * @return true when attaching documents to existing case is successful, otherwise false
      */
     public boolean attach(Envelope envelope, CaseDetails existingCase) {
+        String loggingContext = String.format(
+            "Envelope ID: %s. File name: %s. Case ref: %s. Case state: %s",
+            envelope.id,
+            envelope.zipFileName,
+            existingCase.getId(),
+            existingCase.getState()
+        );
         if (mapper.getDocsToAdd(getDocuments(existingCase), envelope.documents).isEmpty()) {
-            log.warn("Envelope {} has no new documents. CCD Case {} not updated", envelope.id, existingCase.getId());
+            log.warn("Envelope has no new documents. CCD Case not updated. " + loggingContext);
         } else {
-            log.info(
-                "Attaching supplementary evidence. Envelope: {}. Case: {}. Case type: {}",
-                envelope.id,
-                existingCase.getId(),
-                existingCase.getCaseTypeId()
-            );
+            log.info("Attaching supplementary evidence. " + loggingContext);
```

Comment:
shall we use `log.level("Fact. {}", context);` instead?

Reply:
kk